### PR TITLE
[SAT-102][SAT-103] Add analytics, pass analytic_id in route tags

### PR DESCRIPTION
**Pull request by vvfesik**
## Description of changes in release / Impact of release:
- adds heap analytics;
- adds analytic_id to route tags;

### Is this a breaking change?
  - [ ] Yes
  - [ ] No

### If you answered Yes then describe why is it so
(insert text here)

#### Additional details go here
(insert text here)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "postinstall": "npm run build -- --log-level 4",
     "clean": "rm -rf dist/ && rm -rf build/ && rm -rf .cache/",
     "dev": "NODE_ENV=dev SATELLITE_API_PORT=8089 parcel src/index.html --open",
-    "build": "SATELLITE_API_PORT=8089 parcel build src/index.html --no-source-maps",
+    "build": "HEAP_ID=${HEAP_ID} SATELLITE_API_PORT=8089 parcel build src/index.html --no-source-maps",
```
Comment (rozenbom) [inline package.json:27]: What is the reason for "HEAP_ID=${HEAP_ID}"? If we have HEAP_ID defined in the env shouldn't it be accessible already by the config?

File: .circleci/config.yml [hunk 2]
```diff
@@ -153,7 +153,7 @@ jobs:
             . venv/bin/activate
             . "/usr/local/opt/nvm/nvm.sh"
             nvm use 14.15.0
-            npm run electron:build
+            ${HEAP_ID} npm run electron:build
```
Comment (rozenbom) [inline .circleci/config.yml:156]: HEAP_ID is already in the env. Here you just refer it's value which will lead to an error. If we don't want to enable Heap for all the jobs we probably need to:

1. Pick a different name for the variable stored in CircleCI (like CI_HEAP_ID)
2. Here use it as HEAP_ID=${CI_HEAP_ID} npm run electron:build
3. Or alternatively define another config param like HEAP_ENABLED

If we're OK to enable Heap for all the jobs just remove ${HEAP_ID} reference here.

File: package.json (hunk 1 above)
Comment (vvfesik) [reply package.json]: thx for noticing, updated in following commit

File: .circleci/config.yml (hunk 2 above)
Comment (vvfesik) [reply .circleci/config.yml]: thx, updated in following commit

**Review by rozenbom — APPROVED** [review]
LGTM

**Pull request merged**